src/test/testUtil.ts

```diff
@@ -255,12 +255,13 @@ export async function closeAllEditors(): Promise<unknown> {
         return await vscode.commands.executeCommand('openEditors.closeAll')
     }
 
-    // Remove the below code (all of it) when `openEditors.closeAll` is available in all versions
-    await vscode.commands.executeCommand('workbench.action.closeAllEditors')
+    // TODO: Remove the below code (all of it) when `openEditors.closeAll` is available in all versions
 
     // The output channel counts as an editor, but you can't really close that...
     const noVisibleEditor: boolean | undefined = await waitUntil(
```

Comment:
Should we also do this for `openEditors.closeAll` ?

Reply:
Might as well. These editors only show up because a few tests (not sure which) finish without waiting for the editor to open.